rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/tsdb/140_routing_path.yml

```diff
@@ -0,0 +1,207 @@
+missing routing path field:
+  - skip:
+      features: close_to
+
+  - do:
+      indices.create:
+        index: test
+        body:
+          settings:
+            index:
+              mode: time_series
+              routing_path: [uid, tag]
```

Comment:
Then I'm still unsure why this tests fails with the flattened field test.

Reply:
I was using the incorrect field name while doing `addString`. The field name must be the full field name so that the field name is the same both at indexing time and at query time.